src/routes/organizations/organizations.service.ts

```diff
@@ -114,7 +99,7 @@ export class OrganizationsService {
     return await this.organizationsRepository.findOneOrFail({
       where: {
         id,
-        userOrganizations: { user: { id: userId } },
+        userOrganizations: { user: { id: userId, status: 'ACTIVE' } },
```

Comment:
Nit: as this now only returns `ACTIVE` `Organization`s, let's rename it to be more explicit.

Reply:
Agreed, changed in [7c984a6](https://github.com/safe-global/safe-client-gateway/pull/2484/commits/7c984a6922ac4c537919938537314c5599e4a7a7)